app/main.py

```diff
@@ -9,9 +9,8 @@ def __init__(self, name: str, age: int) -> None:
 
 
 def create_person_list(people: list) -> list:
-    new_list = []
-    for person in people:
-        new_list.append(Person(name=person["name"], age=person["age"]))
+    new_list = [Person(name=person["name"],
+                age=person["age"]) for person in people]
```

Comment:
It is hard to read this list comprehention - break it according to the best practices